app/src/examples/SharedElementTransitions/Profiles.tsx

```diff
@@ -40,7 +40,7 @@ const springOptions = {
   damping: 15,
 };
 
-const transition = SharedTransition.custom((values) => {
+const transition = new SharedElementTransition().animation((values) => {
```

Comment:
1. Do you think we could provide an API that omits the `new` keyword? I know we need to create and configure a new object but maybe we could provide some factory object/function.
2. What do you recommend in terms of optimization? Is it possible to implement SETs outside of components? Do we need to recommend memoization with `useMemo` or even enable it by default?
3. Technically, progress animation is also an animation. What about renaming `animation` to `targetAnimation` so we have `targetAnimation` for run-and-forget animations (e.g. `withSpring`) and `progressAnimation` for progress-based animations (e.g. `interpolate`)?